core/model/modx/processors/resource/sort.class.php

```diff
@@ -199,6 +206,17 @@ public function sortContexts() {
         return true;
     }
 
+    public function getActiveTargetMenuindex(int $target) {
```

Comment:
Unfortunately I don't think we can't use scalar type hinting yet - last I heard the target minimum PHP version for 3.x is PHP 5.6, while scalar type hints were introduced in 7.0.

Reply:
Never mind this, we'll bump the requirements to PHP7 for MODX3. Yay, scalar type hints!

Reply:
> Never mind this, we'll bump the requirements to PHP7 for MODX3. Yay, scalar type hints!

Seriously? (• ◡•)

Reply:
Seriously?